apps/prairielearn/src/server.js

```diff
@@ -483,6 +483,7 @@ module.exports.initExpress = function () {
   });
   app.use(require('./middlewares/logResponse')); // defers to end of response
   app.use(require('./middlewares/cors'));
+  app.use(require('./middlewares/content-security-policy').default);
```

Comment:
In my testing I think I put the check around like 154 in server.js, so later in the file should be fine but earlier worked too.